RNWCPP/Desktop.IntegrationTests/RNTesterIntegrationTests.cpp

```diff
@@ -100,11 +100,6 @@ TEST_CLASS(RNTesterIntegrationTests)
     TestComponent("IntegrationTestHarnessTest");
   }
 
-  // Timer tests have been disabled in RN. (See RNTesterIntegrationTests.m)
-  BEGIN_TEST_METHOD_ATTRIBUTE(Timers)
-    TEST_IGNORE()
```

Comment:
?  I assume this PR doesn't fix the timer tests?

Reply:
It does not.
That change must have been introduced by a merge.

I’ll revert.